services/system/Accounts/src/RAccounts.cpp

```diff
@@ -22,8 +22,19 @@ namespace SystemService
          Accounts::Tables tables{Accounts::service};
          return tables.open<AccountTable>().get(account);
       }
+
+      auto getAccounts(std::vector<AccountNumber> accounts) const
+      {
+         std::vector<std::optional<Account>> result;
+         for (auto account : accounts)
+         {
+            Accounts::Tables tables{Accounts::service};
+            result.push_back(tables.open<AccountTable>().get(account));
+         }
+         return result;
+      }
```

Comment:
This reads like a function that takes accounts and returns account.
maybe simply rename the param to account_nums? Anything here to distinguish input frmo output I think would help a ton with readability

Reply:
The thing that devs interact with is the schema, does that seem clearer?

```
type ResourceLimit {
    value: String!
}
type Account {
    accountNum: String!
    authService: String!
    resourceBalance: ResourceLimit
}
type Query {
    getAccount(account: String!): Account
    getAccounts(accounts: [String!]!): [Account]!
}
```

Reply:
Changed parameter names